docs/internals/contributing/writing-code/accessibility.txt

```diff
@@ -0,0 +1,124 @@
+assistive technologies.
+
+.. _accessibility-standards:
+
+Accessibility standards
+=======================
+
+We work to conform with the `Web Content Accessibility Guidelines`_ (WCAG),
+version 2.2, at the AA level. WCAG is the most established standard for web
+accessibility. AA-level guidelines are the most common legal compliance target
+worldwide.
+
+We also attempt to follow other best practices, such as:
+
+* WCAG 2.2 AAA-level guidelines.
+* The upcoming `WCAG 3.0`_ guidelines.
+* `Authoring Tool Accessibility Guidelines`_ (ATAG) 2.0 – guidelines for
+  software and services that are used to produce web content.
+
+To learn more about accessibility without diving straight into standards, we
+recommend `The A11Y Project`_, a community-driven effort to make digital
+accessibility easier.
+
+.. _Web Content Accessibility Guidelines: https://www.w3.org/TR/WCAG22/
+.. _WCAG 3.0: https://www.w3.org/TR/wcag-3.0/
+.. _Authoring Tool Accessibility Guidelines: https://www.w3.org/TR/ATAG20/
+.. _The A11Y Project: https://www.a11yproject.com/
+
+.. _accessibility-support-targets:
+
+Accessibility support targets
+=============================
+
+Beyond standards, we want to make sure Django actually works for people using
+a wide range of input devices and assistive technologies. This has benefits to
+make sure Django works better `for everyone`_. The best way to do this
+effectively is to take accessibility considerations into account as part of
+designing features. If in doubt, consult with users who rely on assistive
+technologies or with accessibility experts.
+
+.. _for everyone: https://en.wikipedia.org/wiki/Curb_cut_effect
+
+.. _accessibility-testing-baseline:
+
+Accessibility testing baseline
+------------------------------
+
+Design the UI with accessibility in mind, and the testing will only be needed
+as a final check. For more complex interfaces, confer with other contributors
+to decide on testing targets. Reach out to the `Accessibility team`_ for
+support and to coordinate testing.
+
+Always test user interface changes with:
+
+* Keyboard-only navigation
+* The `Accessibility Insights`_ browser extension's automated checks feature,
+  or an equivalent tool with the `Axe`_ checker.
+
+Where the UI changes could affect those modalities, also test with:
+
+* Touch-only navigation
+* 400% browser zoom
+* Forced-colors mode (for example Windows Contrast Themes)
+
+.. _assistive-technologies:
+
+Assistive technologies
+----------------------
+
+Where the UI changes could affect assistive technologies, here are popular
+free options we recommend testing with:
+
+* `NVDA`_ on Windows with Firefox ESR
+* `VoiceOver`_ on macOS with Safari
+* `Orca`_ on Linux with Firefox ESR
+* `Narrator`_ on Windows with Microsoft Edge
+* Windows Magnifier and macOS Zoom
+* Windows Speech Recognition and macOS Voice Control
+* VoiceOver on iOS, or TalkBack on Android
+* `Contrast themes`_ on Windows
```

Comment:
```suggestion
Where the UI changes could affect assistive technologies, here are popular free
options we recommend testing with.

For Windows:

* `NVDA`_ - recommended with Firefox ESR
* `Narrator`_ - recommended with Microsoft Edge
* Windows Magnifier
* Windows Speech Recognition
* `Contrast themes`_

For macOS:

* `VoiceOver`_ - recommended with Safari
* macOS Zoom
* macOS Voice Control

For Linux:

* `Orca`_ - recommended with Firefox ESR

For mobile or tablet:

* VoiceOver on iOS, or TalkBack on Android
```
Suggestion: the reader will probably only have access to one OS and so I would group this and I tried to incorporate [this comment](https://github.com/django/django/pull/17338/files#r1487963102)
I am not sure on the "mobile or tablet" group name